app/src/main/java/com/example/trempel/PdpFragment.kt

```diff
@@ -0,0 +1,111 @@
+package com.example.trempel
+
+import android.Manifest
+import android.app.Activity.RESULT_OK
+import android.app.AlertDialog
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.os.Environment
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import java.io.File
+import androidx.core.app.ActivityCompat.startActivityForResult
+
+import android.content.Intent
+import android.media.ImageReader
+import android.net.Uri
+import android.provider.MediaStore
+
+
+class PdpFragment : Fragment() {
+
+    private lateinit var iVProduct: ImageView
```

Comment:
Please, do not use `lateinit` for the properties that you initialize, it should be used only for cases where the initialization is handled by DI